build.sh

```diff
@@ -35,17 +35,6 @@ install_dir=/tmp/scitt
 
 mkdir -p $install_dir
 
-mkdir -p build/attested-fetch
-CC="$CC" CXX="$CXX" \
-    cmake -GNinja -B build/attested-fetch \
-    -DCMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE}" \
-    -DCMAKE_INSTALL_PREFIX=$install_dir \
-    -DCOMPILE_TARGET="${ATTESTED_FETCH_PLATFORM}" \
-    "$root_dir/3rdparty/attested-fetch"
-
-ninja -C build/attested-fetch ${NINJA_FLAGS} --verbose
-ninja -C build/attested-fetch ${NINJA_FLAGS} install
-
 if [ "$PLATFORM" = "sgx" ]; then
     ATTESTED_FETCH_MRENCLAVE_HEX=$(/opt/openenclave/bin/oesign dump -e $install_dir/libafetch.enclave.so.signed | sed -n "s/mrenclave=//p")
```

Comment:
This should go as well as the idea was to register the mrenclave from the attested fetch binary in the ledger application as a trusted enclave that it can communicate with. So the code paths that rely on `ATTESTED_FETCH_MRENCLAVE_HEX` need to be removed as well